### ISSUE TITLE
fc8 without softmax operation

### ISSUE THREAD

**Issue by bruce1408** [stephen-v/tensorflow_alexnet_classify#7]
in alexnet.py file, the last layer is fc8 ,and return fc8 as result , in main_alexlet.py , return alexnet.py result , 
```
in line 107:
fc8 = alexnet(x, keep_prob, num_classes)
```
fc8 without softmax operation, so is it necessary to remove this operation?
